core/string/translation.cpp

```diff
@@ -1300,6 +1333,275 @@ StringName TranslationServer::doc_translate_plural(const StringName &p_message,
 	return p_message_plural;
 }
 
+bool TranslationServer::is_pseudolocalization_enabled() const {
+	return pseudolocalization_enabled;
+}
+
+void TranslationServer::set_pseudolocalization_enabled(bool enabled) {
```

Comment:
Parameters should start with `p_`:
```suggestion
void TranslationServer::set_pseudolocalization_enabled(bool p_enabled) {
```